packages/block-library/src/image/editor.scss

```diff
@@ -124,3 +124,9 @@
 		caption-side: bottom;
 	}
 }
+
+// Edge case to spacing required for images when displayed within Section Block
```

Comment:
Can you expand on more on why this edge case is needed? Ideally blocks shouldn't rely on classnames from other blocks. For example, what happens if the image block is used inside another third-party section block?

Reply:
I've removed the requirement for this. I agree. It was never a good idea.